templates/config/jail.conf.j2

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
+
+{% if elao_fail2ban_configs.ignoreip|default(false) %}
+ignoreip = {{ elao_fail2ban_configs.ignoreip }}
+{% endif %}
+bantime = {{ elao_fail2ban_configs.bantime|default(600) }}
+maxretry = {{ elao_fail2ban_configs.maxretry|default(3) }}
```

Comment:
use _macro to handle extra parameters. (see https://github.com/ElaoInfra/ansible-role-php/blob/master/templates/configs/config_dev.ini.j2)